test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -363,14 +362,18 @@ public void CreateDescriptor_BuildsDescriptorsWithInheritedProperties()
         {
             // Arrange
             var errorSink = new ErrorSink();
-            var intProperty = typeof(InheritedSingleAttributeTagHelper).GetProperty(
-                nameof(InheritedSingleAttributeTagHelper.IntAttribute));
+
+            // Also confirm isStringProperty is calculated correctly.
```

Comment:
Oops, no.  This is about the combination of `TagHelperDescriptorFactory` and `TagHelperAttributeDescriptor` (where I've changed the line between them, moving the calculation to `TagHelperAttributeDescriptor`) getting the property right for _each_ attribute descriptor.